### PR TITLE
Add support Xca9543a and Xca9545a

### PR DESCRIPTION
**Pull request by Atmelfan**
Not yet tested

### CODE REVIEW

**Comment by eldruin** [comment]
Interesting. Thanks for the PR. I will check it in depth soon and let you know.

**Review by eldruin — COMMENTED** [review]
Looks great. The diff got quite messy due to moved code so it was a bit difficult to parse :)
I'll merge it once the small comment is solved and then release a new version.

File: src/lib.rs [hunk 1]
```diff
@@ -211,6 +219,7 @@ pub enum SlaveAddr {
     /// Default slave address
     Default,
     /// Alternative slave address providing bit values for A2, A1 and A0
+    /// Note: Some devices might not have all Ax pins.
```
[the remark this answers is not in the document]
Comment (Atmelfan) [reply src/lib.rs]: I added that unavailable Ax pins should be set to false.
pca954x

[the remark this answers is not in the document]
Comment (eldruin) [reply src/lib.rs]: Great, thanks!

**Comment by eldruin** [comment]
Thanks! This is now merged. Now I am wondering whether to add support for other similar xca954x devices, since there are many.

**Comment by eldruin** [comment]
I have now released version 0.2.1 containing these changes. Sorry this took me a while.

**Pull request merged**